frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -719,15 +719,12 @@ export default ({
       this.$refs.fileAttachmentInputEl.click()
     },
     fileAttachmentHandler (filesList, appendItems = false) {
-      const containsExtension = v => /.+\.\w+$/.test(v) // check if a entry name ends with a file extension.
-      const isDirectory = entry => entry.type === '' && !containsExtension(entry.name)
```

Comment:
Removed this here and the logic to filter out directories has been moved to `drag-end` event listener. (As it's only an issue with drag&drop feature)